types/linkifyjs/test/linkifyjs-tests.ts

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+import * as linkifyjs from "linkifyjs";
+import { JSDOM } from 'jsdom';
```

Comment:
It doesn't look like this dependency is necessary -- you could get an element in another way without using this.